lib/utils/plugins.js

```diff
@@ -69,6 +69,7 @@ const clearModulesCache = () => {
 };
 
 const getPluginName = path => window.require('path').basename(path);
+const getPluginVersion = path => window.require(window.require('path').resolve(path, 'package.json')).version;
```

Comment:
Also I'd import `path` to a variable
```
const path = window.require('path')
```
not really necessary since require caches it but looks cleaner